### DISCUSSION TITLE
BPG web-based interactive tool?

### DISCUSSION THREAD

**Discussion by jarbet** [uclahs-cds/public-R-BoutrosLab-plotting-general#39] [Q&A] [answered]
I'm not sure if this is related to #37 but the original [BPG paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2610-2) abstract says the following:

> A web-based interactive tool allows online figure customization, from which R code can be downloaded for integration with computational pipelines.

Does anyone know the link where I can access this web-based tool?

**Comment by stefaneng** [comment, marked as answer]
@jarbet Those issues are not related. There is a shiny app that worked in Toronto that has not been ported over. There have been a few efforts to revive it:
  - https://github.com/uclahs-cds/project-shipit-BPG_rshiny
  - https://github.com/uclahs-cds/project-ship-it-day-shiny-BPG

there is a very similar app for VennDiagram as well but has similar issues to the BPG shiny app: https://github.com/uclahs-cds/project-R-ShinyVennDiagram
